services/common-threats.yaml

```diff
@@ -32,7 +32,7 @@ threats:
       - TA009
       - T1557
   - id: CCC.TH04
-    title: Attacker encrypts data with client-managed keys
+    title: Attacker encrypts data with client-managed keys #TODO: Should this be in the crypto service?
     description: |
       The service provides encryption mechanisms, but the encryption keys are
```

Comment:
`common_threats.yaml` line 54 has a TODO. I think Multi-region deployment make sense to add there. Better to fix it with this MR as CCC.TH05 (line 48-57) should be included in KMS threats.yaml because it is referred by CCC.C06 which is added in KMS controls.yaml